source/transactions/tests/README.rst

```diff
@@ -342,6 +342,20 @@ fail point on the mongos server which "session0" is pinned to::
               failCommands: ["commitTransaction"]
               closeConnection: true
 
+assertSessionTransactionState
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The "assertSessionTransactionState" operation instructs the test runner to
+assert that the transaction state of the given session is equal to the
+specified value. The possible values are as follows: ``no``, ``starting``,
+``in_progress``, ``committed``, ``aborted``.
```

Comment:
I realize "no" is the result of removing "transaction" from the list of states in the spec, but I think "none" would read better in this instance (as used in the [state diagram](https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#clientsession-changes)).